docs/serving_monitoring/monitor-jh.md

```diff
@@ -1,5 +1,7 @@
 * **How to Monitor JupyterHub workloads using Grafana dashboards** 
-    -  (Video tutorial forthcoming)
+
+[![JH how to](https://img.youtube.com/vi/1zZ-7mdxOWU/0.jpg)](https://youtu.be/1zZ-7mdxOWU)
+
     -  To troubleshoot issues while running your notebooks, you can refer to the [Troubleshooting Runbooks][12]
```

Comment:
This bit is not rendering properly here. 
I think this is an indentation issue.

Reply:
I think I fixed it

Reply:
Thanks :+1: